src/framework/theme/components/checkbox/_checkbox.component.theme.scss

```diff
@@ -6,71 +6,102 @@
 
 @mixin nb-checkbox-theme() {
   @mixin hover-checked-border($color) {
-    .custom-control-input:hover ~ .custom-control-indicator,
-    .custom-control-input:checked ~ .custom-control-indicator,
-    .custom-control-input:hover:checked ~ .custom-control-indicator {
-      border-color: $color;
+    .customised-control-input:hover ~ .customised-control-indicator,
+    .customised-control-input:checked ~ .customisedised-control-indicator,
+    .customised-control-input:hover:checked ~ .customised-control-indicator {
+      //border-color: $color;
+      // TODO implement hover variables
```

Comment:
let's hardcode hover implementation where we agree that hover is same as active but slightly lighter (with scss lighten).

Reply:
Done